test/unit/host_test.rb

```diff
@@ -1921,66 +1923,66 @@ def teardown
     # Ip validations
     test "unmanaged hosts don't require an IP" do
       h=FactoryGirl.build(:host)
-      refute h.require_ip_validation?
+      refute h.require_ip4_validation?
     end
```

Comment:
I'd still exercise `require_ip_validation` (with params) instead. Also make sure you try the IPv6 code flow which isn't exercised at all.